src/app/documents/DocsFormContainer.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
```

Comment:
I'm not sure, but this component looks a lot like the other form container we had (location, service). The only clear differences I can see are the specific routes file and the data fetching action.
Could we not pass those in as props, and reuse 1 component for all these cases?